core-services/image-mirroring/knative/mapping_knative_v0_19_quay

```diff
@@ -22,3 +22,7 @@ registry.ci.openshift.org/openshift/knative-v0.19.1:knative-client quay.io/opens
 registry.ci.openshift.org/openshift/knative-v0.19.1:kn-cli-artifacts quay.io/openshift-knative/kn-cli-artifacts:v0.19.1
 registry.ci.openshift.org/openshift/knative-v0.19.1:knative-client-test-helloworld quay.io/openshift-knative/knative-client-test-helloworld:v0.19.1
 registry.ci.openshift.org/openshift/knative-v0.19.1:knative-client-test-helloworld quay.io/openshift-knative/knative-client-test-grpc-ping:v0.19.1
+registry.ci.openshift.org/openshift/knative-v0.19.1:knative-eventing-kafka-test-kafka-publisher quay.io/openshift-knative/kafka-publisher:v0.19.1
+registry.ci.openshift.org/openshift/knative-v0.19.1:knative-eventing-kafka-test-kafka-publisher quay.io/openshift-knative/kafka-publisher:v0.19
+registry.ci.openshift.org/openshift/knative-v0.19.1:knative-eventing-kafka-test-kafka-performance quay.io/openshift-knative/kafka-performance:v0.19.1
+registry.ci.openshift.org/openshift/knative-v0.19.1:knative-eventing-kafka-test-kafka-performance quay.io/openshift-knative/kafka-performance:v0.19
```

Comment:
It looks like there are more images in the repository which are not mirrored such as knative-eventing-kafka-webhook: https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-v0.19.1__46.yaml#L11
Why are you not mirroring them as well?